src/Propel/Runtime/Adapter/Pdo/PdoStatement.php

```diff
@@ -1,101 +0,0 @@
-<?php
-
-/**
- * This file is part of the Propel package.
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- *
- * @license MIT License
- */
-
-namespace Propel\Runtime\Adapter\Pdo;
-
-use Propel\Runtime\Connection\StatementInterface;
```

Comment:
actually it's used in quite few places, shouldn't it's usages be converted to usages of `\PdoStatement` as well, am I getting You right?